framework/contrib/hit/Makefile

```diff
@@ -21,8 +21,10 @@ endif
 
 ifeq ($(UNAME), Darwin)
 	DYNAMIC_LOOKUP := -undefined dynamic_lookup
+	hit_pad_LDFLAGS := -Wl,-headerpad_max_install_names
```

Comment:
Isn't this only an Intel Mac issue? Arm mac is okay? If so, we should only apply it there. Darwin is both arches here